types/office-js-preview/index.d.ts

```diff
@@ -25058,349 +27701,143 @@ declare namespace Excel {
         readonly onVisibilityChanged: OfficeExtension.EventHandlers<Excel.WorksheetVisibilityChangedEventArgs>;
         /**
         * Overrides the JavaScript `toJSON()` method in order to provide more useful output when an API object is passed to `JSON.stringify()`. (`JSON.stringify`, in turn, calls the `toJSON` method of the object that is passed to it.)
-        * Whereas the original Excel.Worksheet object is an API object, the `toJSON` method returns a plain JavaScript object (typed as `Excel.Interfaces.WorksheetData`) that contains shallow copies of any loaded child properties from the original object.
+        * Whereas the original `Excel.WorksheetCollection` object is an API object, the `toJSON` method returns a plain JavaScript object (typed as `Excel.Interfaces.WorksheetCollectionData`) that contains an "items" array with shallow copies of any loaded properties from the collection's items.
         */
-        toJSON(): Excel.Interfaces.WorksheetData;
+        toJSON(): Excel.Interfaces.WorksheetCollectionData;
     }
     /**
-     * Represents a collection of worksheet objects that are part of the workbook.
+     * Represents the protection of a sheet object.
      *
      * @remarks
-     * [Api set: ExcelApi 1.1]
+     * [Api set: ExcelApi 1.2]
      */
-    class WorksheetCollection extends OfficeExtension.ClientObject {
+    class WorksheetProtection extends OfficeExtension.ClientObject {
         /** The request context associated with the object. This connects the add-in's process to the Office host application's process. */
         context: RequestContext;
-        /** Gets the loaded child items in this collection. */
-        readonly items: Excel.Worksheet[];
         /**
-         * Adds a new worksheet to the workbook. The worksheet will be added at the end of existing worksheets. If you wish to activate the newly added worksheet, call `.activate()` on it.
-         *
-         * @remarks
-         * [Api set: ExcelApi 1.1]
-         *
-         * @param name Optional. The name of the worksheet to be added. If specified, the name should be unique. If not specified, Excel determines the name of the new worksheet.
-         */
-        add(name?: string): Excel.Worksheet;
-        /**
-         * Inserts the specified worksheets of a workbook into the current workbook.
-                    
-                     **Note**: This API is currently only supported for Office on Windows and Mac. And it has been deprecated, please use `Workbook.insertWorksheetFromBase64` instead.
+         * Specifies the `AllowEditRangeCollection` found in this worksheet.
```

Comment:
Here's another place it'd be great to elaborate on the value and purpose of this API.